staging/src/k8s.io/apiserver/pkg/endpoints/filters/audit_test.go

```diff
@@ -436,12 +436,13 @@ func TestAuditJson(t *testing.T) {
 	delay := 500 * time.Millisecond
 
 	for _, test := range []struct {
-		desc     string
-		path     string
-		verb     string
-		auditID  string
-		handler  func(http.ResponseWriter, *http.Request)
-		expected []auditv1alpha1.Event
+		desc       string
+		path       string
+		verb       string
+		auditID    string
+		handler    func(http.ResponseWriter, *http.Request)
+		expected   []auditv1alpha1.Event
+		respHeader bool
```

Comment:
Because some test cases don't call http.ResponseWriter.WriteHeader() function. So we can't check the response http header.